mindocr/data/builder.py

```diff
@@ -117,13 +120,22 @@ def build_dataset(
     ds = ms.dataset.GeneratorDataset(
                     dataset,
                     column_names=dataset_column_names,
-                    num_parallel_workers=num_workers,
+                    num_parallel_workers=1,
```

Comment:
You can try num_parallel_workers=4, 8,12. It should be faster.

Reply:
Found that when num_parallel_workers > 1, the speed is much lower. I guess it is due to reason that the dataset generator （based on LMDB)  uses multiple thread to read the database already (which corresponding to the configure `max_readers` in the LMDB python-binding). So without change the LMDB configuration, num_workers=1 seems to be the best choice.